assets/javascripts/kitten/components/cards/contribution-card/components/action.js

```diff
@@ -1,17 +1,38 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import classNames from 'classnames'
 import { Button } from '../../../../components/buttons/button'
+import { CheckedCircleIcon } from '../../../icons/checked-circle-icon'
+import COLORS from '../../../../constants/colors-config'
+
+export const Action = ({ isInputValid, ...props }) => {
 
-export const Action = props => {
   return (
-    <Button
-      borderRadius={4}
-      type="submit"
-      {...props}
-      className={classNames(
-        'k-ContributionCard__action',
-        props.className,
+    <>
+      <Button
+        borderRadius={4}
+        type="submit"
+        {...props}
+        className={classNames(
+          'k-ContributionCard__action',
+          { 'k-ContributionCard__actionHide': isInputValid },
```

Comment:
peut-être qu'on peut aller un peu plus simplement

```suggestion
          { 'k-u-hidden': isInputValid },
```